src/surfing_penguin/API/User.py

```diff
@@ -1,10 +1,10 @@
 """routes.py: Each function in this file indicates a web page (HTML page)."""
 from src.surfing_penguin import surfing_penguin
-from src.surfing_penguin.extensions import login_manager
 from src.surfing_penguin.routes import api
 from flask_restplus import Resource, fields
+from functools import wraps
 from src.surfing_penguin.db_interface import UserFunctions
-from flask_login import login_user, logout_user, current_user, login_required
+from flask_login import login_user, logout_user, current_user
 
```

Comment:
A good convention is that build-in modules would be imported first, before 3rd-party modules and one's own modules.